weatherrouting/routers/linearbestisorouter.py

```diff
@@ -50,13 +50,21 @@ def generate_path(p):
             path = path[::-1]
             position = path[-1].pos
 
-        if self.grib.getWindAt(time + datetime.timedelta(hours=1), end[0], end[1]):
+        if self.grib.getWindAt(
+            time + datetime.timedelta(hours=timedelta), end[0], end[1]
```

Comment:
I guess that was the case before but it seems we have a time offset of `timedelta`, as log show first wind request is made with `startDateTime` + `timedelta`.

Reply:
Can you further detail what do you mean?